### PR TITLE
Prepare release 1.51.0

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ Here are the release managers for future versions with the tentative release dat
 
 | Version | Release Manager | Tentative release date |
 |---------|-----------------|------------------------|
-| 1.51.0  | @yurishkuro     | 5 November 2023        |
 | 1.52.0  | @jkowall        | 5 December 2023        |
-| 1.53.0  | @pavolloffay    | 3 January 2023         |
-| 1.53.0  | @joe-elliott    | 7 February 2023        |
-| 1.54.0  | @albertteoh     | 6 March 2023           |
+| 1.53.0  | @pavolloffay    | 3 January 2024         |
```
Comment (albertteoh) [inline RELEASE.md:58]: Nice catch! Caught us napping 😅

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,23 +1,81 @@
...
+
+#### 🚧 Experimental Features:
+
+* Exp(jaeger-v2): simplify all-in-one configuration ([@yurishkuro](https://github.com/yurishkuro) in [#4875](https://github.com/jaegertracing/jaeger/pull/4875))
+* Exp: support primary and archive storage ([@yurishkuro](https://github.com/yurishkuro) in [#4873](https://github.com/jaegertracing/jaeger/pull/4873))
+* Feat(jaeger-v2): create default config for all-in-one ([@yurishkuro](https://github.com/yurishkuro) in [#4842](https://github.com/jaegertracing/jaeger/pull/4842))
+
+#### 👷 CI Improvements:
+
+* Ci: split the install-tools into test/build groups ([@MeenuyD](https://github.com/MeenuyD) in [#4878](https://github.com/jaegertracing/jaeger/pull/4878))
```
Comment (albertteoh) [inline CHANGELOG.md:52]: The `Ci: ` prefixes seem a little superfluous, and similarly for other sections.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,23 +1,81 @@
...
+<details>
+<summary>next release template</summary>
+
 next release (yyyy-mm-dd)
 -------------------
 
 ### Backend Changes
 
-#### ⛔ Breaking Changes
+run `make changelog` to generate content
```
Comment (albertteoh) [inline CHANGELOG.md:16]: What is the motivation for adding this? It's already documented in RELEASE.md.

**Review by albertteoh — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (yurishkuro) [reply CHANGELOG.md:16]: It was distracting / confusing having empty sections especially after copying the template, and there were many sections that are no longer necessary.

File: CHANGELOG.md (hunk 2 above)
Comment (yurishkuro) [reply CHANGELOG.md:52]: Yeah, although not unusual in other projects. Now that we switched to using labels we should pay closer attention to PR titles to be exactly what we want to show up in the changelog.

**Pull request merged**